agent/util-scripts/pbench-register-tool

```diff
@@ -116,7 +116,7 @@ function usage() {
 	printf -- "\tdenoted by a leading hash, or pound (\"#\"), character.\n"
 	printf -- "\nAvailable tools:\n"
 	local tool=""
-	for tool in $(find ${pbench_bin}/tool-scripts -maxdepth 1 ! -type d ! -name '*README*' ! -name base-tool ! -name unittests -printf "%P\n" 2> /dev/null | sort); do
+	for tool in $(find ${pbench_bin}/tool-scripts -maxdepth 1 ! -type d ! -name '*README*' ! -name base-tool ! -name meta.json ! -name unittests -printf "%P\n" 2> /dev/null | sort); do
```

Comment:
We want to eventually replace this `for` loop with an internal CLI call which will emit the list of tools available as derived from the `meta.json` file.

Is this something we should do in this PR or wait until later?  I would like to see this done as part of this PR, but open to other opinions.